dashboard/app/models/level.rb

```diff
@@ -186,6 +186,20 @@ def self.key_to_params(key)
     end
   end
 
+  # Returns true if this is a pixelation level.
+  def pixelation?
+    # TODO(dave|joshlory): Define Pixelation < DSLDefined as a new level type,
+    # eliminating this fragile test of 'href'.
+    self.is_a?(DSLDefined) && self.try(:properties).try(:[], "href") == "pixelation/pixelation.html"
```

Comment:
I'm surprised we need `self.try(:properties)` here.  Is there a situation where `properties['href']` could throw?